lensed_hosts/generate_lensed_hosts_agn.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python
+import numpy as np
+import os
+import argparse
+import pylab as pl
+import subprocess as sp
+import astropy.io.fits as pyfits
+import pandas as pd
+import scipy.special as ss
+import om10_lensing_equations as ole
+import sqlite3 as sql
+import sys
+
+#data_dir = os.path.join(os.environ['SIMS_GCRCATSIMINTERFACE_DIR'], 'data')
+data_dir = 'data/new_files/'
```

Comment:
Can you remove the truth databases from the PR and change this to be a command line argument that specifies the data directory you want to use?

Reply:
I was thinking about this some more. Maybe we can include a script to download the truth catalogs from somewhere public facing so we don't have to have them in the repository but they are still easy to get so this code can be run without too much setup. What do you think?

Reply:
@jbkalmbach why do we want to avoid having the truth catalogs in the repository?

Reply:
I think that the plan is to have all DC2 truth catalogs in one place on cori. This includes the static truth catalogs along with the sprinkled truth catalogs so we would want to be able to specify that location when running this code.

Reply:
@jbkalmbach I never answered your first question: I have looked at simsGCRCatSimInterface and `InstanceCatalogWriter.py` still can include catalogs of lensed hosts in the instance catalogs it generates.  With regard to how to locate SNe, that is included in the SNe lensing program.  Nan wrote code to place the supernova at a random location within the galaxy.  That code is in fact the only real difference between the AGN lensing program and the SNe lensing program.